### PR TITLE
adding two new static pages

### LINKED ISSUE #1628: New Static Page for Community Curation
Please can we have a new static page for community curation at /communitycuration and that has the text contained within this document: https://docs.google.com/document/d/19zKssvhfEMxPuTMseDuKPLgi6piFvqmy9TYhuUtVRK0/edit?usp=sharing

Please let me know if you have any questions. Thank you!

### PR DESCRIPTION
**Pull request by ramGranell**
This is related to the ticket  #1628 . The URLs are 
/community_curation -> https://deploy-preview-1631--fairsharing.netlify.app/community_curation

### CODE REVIEW

File: src/data/communityCurationData.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        }
+      ]
+    },
+    "fourth_section":
+    {
+      "main_title": "How do I get started?",
+      "text": "Make sure you have an account with us ()<a href='https://fairsharing.gitbook.io/fairsharing/' class='underline-effect' target=blank>here's how</a>). Then <a href='mailto:contact@fairsharing.org'>email us</a> with your account information and let us know your domain of interest. We’re looking to engage with Communities across all of the Natural Sciences, Engineering Sciences and Humanities & Social Sciences. We'll be in touch and, if successful, we’ll get you set up with the proper roles. "
```
Comment (knirirr) [inline src/data/communityCurationData.json]:
```suggestion
      "text": "Make sure you have an account with us (<a href='https://fairsharing.gitbook.io/fairsharing/' class='underline-effect' target=blank>here's how</a>). Then <a href='mailto:contact@fairsharing.org'>email us</a> with your account information and let us know your domain of interest. We’re looking to engage with Communities across all of the Natural Sciences, Engineering Sciences and Humanities & Social Sciences. We'll be in touch and, if successful, we’ll get you set up with the proper roles. "
```

File: src/views/Static/CommunityCuration/CommunityCuration.vue [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                <a :href="props.item.url_organisation">
+                  {{ props.item.organisation }}
+                </a>
+              </div>
+            </td>
+            <td>{{ props.item.scope }}</td>
+          </tr>
+        </template>
+      </v-data-table>
+    </div>
```
Comment (knirirr) [inline src/views/Static/CommunityCuration/CommunityCuration.vue:58]:
```suggestion
    </div>
    <br/>
```

**Comment by knirirr** [comment]
The pages look OK to me, but are they meant to be linked from anywhere, e.g. the main page footer?

**Review by allysonlister — CHANGES_REQUESTED** [review]
Thank you very much! Looks really nice. Mainly due to seeing this page live, I have some changes - thank you for your patience...!

1. Please can we add a blue banner to this page as there is in /API_doc? The title should read: "FAIRsharing Community Curation Programme" and the tagline should read "Put your expertise into action and get credited by joining our curation programme". This is now added to the gdoc for completeness.
2. There is a typo in the GBIF organisation links in the table - it should ready 1166, not 1666
3. I accidentally put the PNNL link for the static page as the actual PNNL homepage, but it should be https://fairsharing.org/organisations/2284 - I've updated the google doc accordingly, please could you update the static page?
4. Please could you add a little orcid logo (like we have for maintainers on record pages) next to each of our curators' names? I've added their ORCID links to the gdoc.
5. Seeing the questions in the titles as well as the text formatted nicely, I think I can write it more better / more positively, so I've updated the title text as well as some of the paragraph text, so **you'll need to grab all of the text from the google doc again, my apologies**. 
6. Susanna has asked that we change the format of the text below the curator table. She would like to a) move the code of conduct to this static page (so we only have one static page), and then b) make the text below the table collapsible in the same way as is currently done for the FAQ at /educational . This means that below the table of community curators, there will be three collapsible sections: "The Role of Community Curators", "Getting Started", and "Code of Conduct". This will allow us to add more documentation to this page as needed in future without making the page look huge.
7. Please note that the end of the code of conduct text includes some notes I made :-) I've modified the gdoc at the end of the code of conduct section to make it clearer where to stop the text. 

Thank you!

**Comment by allysonlister** [comment]
> The pages look OK to me, but are they meant to be linked from anywhere, e.g. the main page footer?

Good point. I will ask and get back to you. thanks!

**Review by allysonlister — CHANGES_REQUESTED** [review]
Thank you! Just a couple of things please :-)

1. Please could you fix the link here?
heres
2. As mentioned in my last comment, everything below the curator table in this page should be collapsible in the style of the FAQ on the /educational page. There should be three collapsible sections: "The Role of Community Curators", "Getting Started" (please rename the Engagement section to "Getting Started" it's only a sentence or two but should have its own section), and "Code of Conduct".

Thank you!

**Comment by allysonlister** [comment]
> > The pages look OK to me, but are they meant to be linked from anywhere, e.g. the main page footer?
> 
> Good point. I will ask and get back to you. thanks!

Thank you, it will need to go in the footer, however as that will make the right-hand column quite long, it's likely to result in a reshuffle, so I'll make a separate ticket once we've thought about it. In the meantime, we can progress with community_curation, even if it means there is no link to it for a week or so from elsewhere on the site.

**Comment by ramGranell** [comment]
@allysonlister I think I made all the requested changes, but tell me if there is anything missing.

**Comment by allysonlister** [comment]
Thank you - just a final couple of things - you're very patient with me!

1. change "uss" to "us"
uss

2. Now that the sections are collapsible (thank you!) the header size looks too large. Please could you make the headers a bit smaller, e.g. the same size as the internal headers within the "role" section?

Thank you!

**Comment by ramGranell** [comment]
>     2. Now that the sections are collapsible (thank you!) the header size looks too large. Please could you make the headers a bit smaller, e.g. the same size as the internal headers within the "role" section?

The problem with this is that the main title of the section will be the same size than the secondary titles inside the "The Role of Community Curators" section. I made the changes (please check it again) and put them in bold to differentiate them, but I can put them in normal colour. Are you happy with this? Do you want I change as well the title of "Our Community Curators "?

**Review by allysonlister — APPROVED** [review]
Thank you - it looks great Ramon!!

**Review by knirirr — APPROVED** [review]

**Pull request merged**